src/main/java/io/ebeaninternal/server/querydefn/DefaultOrmQuery.java

```diff
@@ -1220,6 +1221,10 @@ public void findEach(Consumer<T> consumer) {
   @Override
   @SuppressWarnings("unchecked")
   public <A> List<A> findSingleAttributeList() {
+    if (!detail.hasSelectClause()) {
+      // (no explicit select set - clear all properties)
+      detail.setBase(new OrmQueryProperties(null, new LinkedHashSet<>()));
+    }
     return (List<A>) server.findSingleAttributeList(this, null);
```

Comment:
if no explicit select clause is set, ebean selects all properties somewhere later - which messes up the single-attribute-query when you want explicitly **fetch**  a property deeper in the object graph. This is not nice, but this is the minimal invasive solution.